.gitignore

```diff
@@ -151,3 +151,6 @@ Pipfile.lock
 
 # pyright
 pyrightconfig.json
+
+# persist dir for chromadb test
+/data/
```

Comment:
you could remove this. guess this is only for your own local testing

Reply:
https://github.com/run-llama/llama_index/blob/90f62e1da1131876b0bdbc3a3d07c785424ac70c/tests/vector_stores/test_chromadb.py#L59
At this line, I define persist_dir as "./data".